tests/TestCase/I18n/TimeTest.php

```diff
@@ -482,31 +544,96 @@ public function testI18nFormat($class)
     }
 
     /**
+     * test formatting dates taking in account default output timezones.
+     *
+     * @dataProvider classNameProvider
+     * @return void
+     */
+    public function testI18nFormatWithDefaultOutputTimezone($class)
+    {
+        $time = new $class('Thu Jan 14 13:59:28 2010');
+
+        $class::setDefaultLocale('en-US');
+        $class::setDefaultOutputTimezone('America/Vancouver');
+
+        $result = $time->i18nFormat();
+        $expected = '1/14/10 5:59 AM';
+        $this->assertTimeFormat($expected, $result);
+
+        $result = $time->i18nFormat(null, 'America/Toronto');
+        $expected = '1/14/10 8:59 AM';
+        $this->assertTimeFormat($expected, $result);
+
+
+        $class::setDefaultLocale('de-DE');
+        $class::setDefaultOutputTimezone('Europe/Berlin');
+
+        $result = $time->i18nFormat();
+        $expected = '14.01.10 14:59';
+        $this->assertTimeFormat($expected, $result);
+
+        $result = $time->i18nFormat(null, 'Europe/London');
+        $expected = '14.01.10 13:59';
+        $this->assertTimeFormat($expected, $result);
+    }
+
+    /**
      * test formatting dates with offset style timezone
      *
      * @dataProvider classNameProvider
-     * @see https://github.com/facebook/hhvm/issues/3637
      * @return void
+     * @see https://github.com/facebook/hhvm/issues/3637
      */
     public function testI18nFormatWithOffsetTimezone($class)
     {
+        // Default output format is in UTC
         $time = new $class('2014-01-01T00:00:00+00');
         $result = $time->i18nFormat(\IntlDateFormatter::FULL);
         $expected = 'Wednesday January 1 2014 12:00:00 AM GMT';
         $this->assertTimeFormat($expected, $result);
 
+        $class::setDefaultOutputTimezone('GMT+09:00');
         $time = new $class('2014-01-01T00:00:00+09');
```

Comment:
Well @markstory this was the key issue with my tests failing. I assumed that there should be a conversion of the offset to the output timezone. I don't use offsets personally and I am okay with this if you are but my gut tells me it should be different (either replacing the timezone or calculating the difference and applying that).

Thanks for making it work(tm)! :)